recipes/rubinenv/meta.yaml

```diff
@@ -0,0 +1,137 @@
+{% set name = "rubinenv" %}
+{% set version = "0.1.0" %}
+{% set build_number = 1 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  skip: True         # [python_impl == 'pypy' or win]
+  number: {{ build_number }}
+
+# Since all requirements are required "at run time" for the
+# rubin developers, all requirements are fundamentally
+# run requirements. However, we've attempted to organize 
+# requirements as build-like, host-like, or run-like, in 
+# order inform any evolution of this recipe.
+requirements:
+  # conda-forge-pinning dependencies should also be in host
+  # https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml
+  host:
+    - boost =1.70
+    - cfitsio
+    - mpich =3.2.1
+    - python =3.7
+  # We need them as host so we know what's installed
+  run:
+    # build-like dependencies
+    # As part of our scons build process, we do run tests, so
+    # these really are required to perform a build.
+    # Note that "testing" in conda-forge should be limited testing 
+    # sucessful packaging, and not testing code.
+    - compilers
+    - cmake
+    - doxygen
+    - flake8 <3.8
+    - make
+    - pep8-naming
+    - pip
+    - psutil
+    - pytest <6
+    - pytest-cov
+    - pytest-doctestplus !=0.7.0
+    - pytest-flake8
+    - pytest-openfiles
+    - pytest-runner
+    - pytest-session2file !=0.1.10
+    - pytest-subtests
+    - pytest-xdist
+    - scons
+    - sysroot_linux-64 =2.17  # [linux64]
+    # host-like and run-like
+    # These are mostly run-like, but for the same reasons as
+    # above, we need most of these at build time, so they are mostly
+    # host-like
+    - boost =1.70
+    - cfitsio
+    - eigen
+    - galsim
+    - libaprutil
+    - lapack
```

Comment:
You should not be building against this package. It is deprecated and has not been updated in over a year. The current version is 3.8.X and this is at 3.6.X. See the documentation on blas.

Reply:
Removing it, I think there was a header in there I needed not in the other packages, but it's find to remove for now and I'll investigate.